CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 cookbook-rb-ips CHANGELOG
 ===============
 
+## 0.4.0
+
+  - David Vanhoucke
+    - [2bdb85f] fix lint
```

Comment:
```suggestion
    - [2bdb85f] fix lint
```
This line could be ommited; it's not necessary to indicate fixes in lint inside the changelog